libraries/botbuilder-dialogs-adaptive-testing/tests/resources/SettingsStateTests/SettingsStateTests_SettingsTest.test.dialog

```diff
@@ -10,14 +10,6 @@
                     {
                         "$kind": "Microsoft.SendActivity",
                         "activity": "${settings.MicrosoftAppId}"
-                    },
-                    {
-                        "$kind": "Microsoft.SendActivity",
-                        "activity": "${settings.MicrosoftAppPassword}"
-                    },
-                    {
-                        "$kind": "Microsoft.SendActivity",
-                        "activity": "${settings.ApplicationInsightsInstrumentationKey}"

```

Comment:
In [.NET, this test was adjusted to ensure these settings are blocked](https://github.com/microsoft/botbuilder-dotnet/pull/5641/files#diff-f1ca39c30c78da07bd745b2ac47e01e2c33b5cb02e96c49aaeec0c476c1ca2acR21). Can we do the same thing, here?

Reply:
Added by the latest commit